doc/source/science_guide/sg_thermo.rst

```diff
@@ -692,6 +695,183 @@ same mean pond area in a grid cell after the addition of new ice,
+drainage), drainage through macro-flaws in the ice (super-cm scale), an
+ice freeboard constraint, drainage during ice deformation, and pond lid
+refreezing. Meltwater is also lost when the ice melts. Unlike in the
+level or topo schemes, the sealvl scheme does not use the 'runoff'
+(``rfrac``) parameterization. Physically, runoff is the same as drainage
+through flaws in the ice. So it is handled by the macro-scale drainage.
+
+* *Percolation Drainage.* Percolation drainage implemented in the mushy 
+  thermodynamics scheme. The harmonic mean of the permeability of the 
+  ice column is estimated, as is the hydraulic head (the height of the 
+  pond-air interface above sea level, see above). Then the drainage rate
+  is estimated assuming a Darcy flow. Percolation drainage in the sealvl
+  scheme is identical to the level scheme except for the calculation of 
+  the hydraulic head.
+
+* *Macro-Flaw Drainage.* Melt water is transported laterally and drains 
+  through macro-flaws: cracks, floe edges, enlarged brine channels, 
+  seal holes, etc... (:cite:`Eicken04`, :cite:`Polashenski12`). In the 
+  real system, the efficiency of this process depends on the 
+  connectivity of lateral flow networks and the frequency of 
+  macro-flaws, both of which evolve with ice conditions. In the sealvl 
+  scheme, macro-flaw drainage is parameterized as an exponential decay 
+  of pond height relative to sea level (a.k.a., the hydraulic head). So 
+  macro-flaw drainage cannot remove pond water that sits below sea 
+  level. The level pond scheme is identical except that the exponential 
+  decay is applied to the entire pond height. The decay constant is 
+  controlled by the ``tscale_pnd_drain`` namelist parameter. Currently, 
+  this decay constant is uniform in time and space, but future work 
+  should consider how changing ice conditions impact macro-flaw 
+  drainage.
+
+* *Ice Freeboard Constraint.* For free-floating ice, pond water cannot 
+  depress the mean ice surface below sea level when there are efficient 
+  water transport pathways (i.e., Stage III melt ponds). The buoyancy 
+  force from the ice drives the redistribution of water from above the 
+  ice to below. Below-sea level pond bottoms are sustained by the weight
+  of adjacent ice and snow above sea level. The sealvl scheme assumes 
+  that each ice category is rigid and mechanically uncoupled from the 
+  other categories. If necessary, pond water is drained such that the 
+  mean ice surface of the category is at sea level. I.e., the mean 
+  category ice freeboard is constrained to be greater than or equal to
+  zero. The level pond scheme has the same constraint, except in the 
+  level pond scheme the ponded area of the category is assumed to be 
+  mechanically uncoupled from the surrounding ice. So in the level pond
+  scheme, the freeboard constrains pond depth to be no greater than 10%
+  of the category ice thickness.
```

Comment:
I suggest removing the last sentence here, since 10% is an approximate value that depends on the seawater, freshwater and ice density parameter values.